Makefile

```
@@ -180,30 +186,37 @@ serve-selenium-notebooks: ## Serve testing notebooks for Jupyter
 	cd lib && export PYTHONPATH=`pwd`; jupyter notebook --notebook-dir=galaxy_test/selenium/jupyter
 
 # Release Targets
-release-create-rc: release-ensure-upstream ## Create a release-candidate branch
+release-create-rc: release-ensure-upstream release-ensure-packaging ## Create a release-candidate branch
+	git diff --quiet
```

```suggestion
	git diff --quiet || echo "Error: some files have changes" && exit 1
```

Maybe this could be another make target, since it's reused below.